<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Qtdelete</class>
 <widget class="QDialog" name="Qtdelete">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>删除</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>数据删除系统</string>
   </property>
  </widget>
  <widget class="QToolBox" name="toolBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>91</width>
     <height>191</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <attribute name="label">
     <string>   删除部门</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="page_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <attribute name="label">
     <string>   删除工种</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="page_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <attribute name="label">
     <string>   删除员工</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="page_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <attribute name="label">
     <string>删除考勤信息</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="page_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <attribute name="label">
     <string>删除加班信息</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>221</width>
     <height>191</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>4</number>
   </property>
   <widget class="QWidget" name="page_6">
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>部门编号</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>70</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_7">
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>工种编号</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>70</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_8">
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>员工编号</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>70</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_9">
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>员工编号</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>考勤时间</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>40</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateEdit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>100</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_10">
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>员工编号</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>加班时间</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_6">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>40</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateEdit_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>100</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
